TIG/TIG/Feature/Home/More/WeeklyRepeatView.swift

```diff
@@ -38,6 +38,7 @@ enum Day: Int, CaseIterable {
 
 struct WeeklyRepeatView: View {
     
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
```

Comment:
```suggestion
    @Environment(\.dismiss) var dismiss
```
위와 같이 간단하게 dismiss에 대한 환경변수를 가져올 수 있습니다.

Reply:
수정완료!